lib/IDE/Utils.cpp

```diff
@@ -607,7 +607,7 @@ accept(SourceManager &SM, SourceLoc Loc, StringRef Text,
 void swift::ide::SourceEditConsumer::
 accept(SourceManager &SM, CharSourceRange Range, StringRef Text,
        ArrayRef<NoteRegion> SubRegions) {
-  accept(SM, RegionType::ActiveCode, {{Range, Text, SubRegions}});
+  accept(SM, RegionType::ActiveCode, {{{}, Range, {}, Text, SubRegions}});
```

Comment:
```suggestion
  accept(SM, RegionType::ActiveCode, {{/*Path=*/{}, Range, /*BufferName=*/{}, Text, SubRegions}});
```
(Ditto for other usages, but I know we already have `Replacements.push_back({LabelRange, Text, {}});` etc.)

Reply:
Will do